docs/src/main/paradox/features/linking.md

```diff
@@ -1,5 +1,4 @@
-Linking
--------
+# Linking
```

Comment:
Yes. Maybe because I'm using Pamflet that can generate combined pages and PDF, I think of the whole doc as one big markdown file.